### ISSUE TITLE
dcos example seems broken

### ISSUE THREAD

**Issue by raanand-home** [Azure/acs-engine#132]
I am pretty new here 
following the instruction and for some reason the template failed.
{
  "error": {
    "code": "PropertyChangeNotAllowed",
    "target": "customData",
    "message": "Changing property 'customData' is not allowed."
  }
}

attached template files
[ocdc.zip](https://github.com/Azure/acs-engine/files/622946/ocdc.zip)
azure portal errors
![image](https://cloud.githubusercontent.com/assets/6960512/20770392/9cbe0338-b756-11e6-8711-87164a0a9b08.png)

**Comment by colemickens** [comment]
Please provide more details about what you did. It looks like you deployed a cluster and then tried to change it and redeploy? Is that the case? If so, please describe exactly what steps you took.

**Comment by anhowe** [comment]
VMs and VMSS does not allow modification of customdata.  For VMSS you need to delete the agent pool, and redeploy, or for AS (VM deployments) you need to delete the VMs and redeploy.  Closing since this is won't fix.

**Comment by smifun** [comment]
I have similar problem. I lost original azuredeploy.json file  that I used for setting up cluster. Now I want to scale up that cluster. How can I get that original customData value that was used ? I found it on node at those locations:
/var/lib/waagent/ovf-env.xml
/var/lib/waagent/CustomData
/var/lib/cloud/instance/user-data.txt

but it has variables resolved. Can it be still used ? Is there any other way ?
